### PR TITLE
CI: use GitHub Actions

### PR DESCRIPTION
**Pull request by thaJeztah**
Travis CI (.org) is no more, so adding a rudimentary action

### CODE REVIEW

**Comment by thaJeztah** [comment]
Example run of CI can be seen on my fork; https://github.com/thaJeztah/go-plugins-helpers/pull/1 (github actions doesn't run here until this is merged).

It's still rudimentary, but we can update once https://github.com/docker/go-plugins-helpers/pull/127 (or an equivalent) is merged, and with this at least we have "some" CI running (equivalent of what travis was running)


@fredericdalleau @ebriney ptal

File: .github/workflows/ci.yaml
Comment (dave-tucker) [inline .github/workflows/ci.yaml:5]: we don't have a main branch, it doesn't hurt to have this just in case we decide to rename it

**Review by dave-tucker — COMMENTED** [review]
Thanks for doing this @thaJeztah. Do you know of an action that could handle DCO checking?
It seems that Gordon may no longer be doing it for this repo as the status is still pending :thinking:

File: .github/workflows/ci.yaml
Comment (thaJeztah) [reply .github/workflows/ci.yaml:5]: Yeah, that was my (lazy) thinking 😂

**Comment by dave-tucker** [comment]
@thaJeztah after this merges, someone from the Docker org with `Admin` access to this repo will need to adjust the required statuses in the branch protection settings too

**Comment by thaJeztah** [comment]
> Do you know of an action that could handle DCO checking?

Oh; hm.. I thought we had the DCO bot enabled on this repo, but I _think_ it may not be running because that has a bug where marking it as "required" while there's existing PRs makes it never complete.

I know containerd also has a bare-bones "action" for DCO check, but overall the official DCO bot should be fine.

Let me check if I have admin access

**Comment by thaJeztah** [comment]
I requested the DCO bot to be added

**Comment by thaJeztah** [comment]
ok; looks to be working now:

Screenshot 2021-06-23 at 11 33 04

**Review by dave-tucker — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
Thanks for reviewing! Looks like you're no longer marked as a "maintainer" (I'll look into that if you want to continue helping out). Let me use my "admin powers"  to merge this one.

(it's still a bit quirky, but I wanted to have at least _some_ CI running 😅)

**Pull request merged**
